_schemaV2.graphql

```diff
@@ -320,7 +321,6 @@ type Alert {
     period: String
     periods: [String]
     priceRange: String
-    published: Boolean
```

Comment:
We likely need to re-add this functionality to filter for _just_ unpublished works - but as Gravity doesn't support it - eh, let's just remove it to keep things logical.

After this PR is merged, we support:
  - returning both published/unpublished works together - via `includeUnpublished: true` - this is for listings page with no filters applied
  - we always support returning just published works - by default, dont include `includeUnpublished: true`

Still to do (requiring minor gravity and MP PR's):
  - actually support returning _just_ unpublished works.

For that - I'll re-add this `published` boolean after its supported in Gravity.